### DISCUSSION TITLE
Stacked group fullscreen question.

### DISCUSSION THREAD

**Discussion by izmyname** [hyprwm/Hyprland#8369] [Q&A] [answered]
Recently, I made the issue for  waybar https://github.com/Alexays/Waybar/issues/3746

tl;dr: waybar's window module doesn't change its colour, if a stacked window group is on fullscreen, unlike non-grouped windows. Could it be the problem on Hyprland's end, like stacked group doesn't report fullscreen condition or smth? 

**UPD**: fixed on latest waybar git

**Comment by Aqa-Ib** [comment]
Looking at hyprctl clients

**Reply by izmyname** [reply]
hyprctl clients reports, that only focused window in the group is on fullscreen, while the rest - aren't.

Should I close waybar issue and open one on Hyprland tracker, in this case, or this discussion would be enough?

**Reply by Aqa-Ib** [reply, marked as answer]
its weird that it doesn't report it as fullscreen because hyprctl clients shows that the active group member is fullscreen indeed, so waybar should report it as fullscreen. Seems to me like a waybar bug.

**Reply by izmyname** [reply]
I see. Thanks for the response.

**Reply by Aqa-Ib** [reply]
I am gonna look at waybar code to see if I find something odd though.

**Reply by Aqa-Ib** [reply]
How are you fullscreening the window? Try with `fullscreen, 1` to see if it works as expected that way.

**Reply by izmyname** [reply]
By using keybind    `bind = $mainMod, F, fullscreen, 1`

**Reply by izmyname** [reply]
Also, fullscreen, 0 has the same problem.

**Reply by Aqa-Ib** [reply]
yes, I have identified the problem on waybar's code, I am gonna post in the waybar issue thread.

**Reply by izmyname** [reply]
Thank you!

**Reply by Aqa-Ib** [reply]
you are welcome :-)

**Reply by Aqa-Ib** [reply]
Try compiling waybar after removing this whole block, cause it seems wrong to me:
https://github.com/Alexays/Waybar/blob/5f260516af9a3e5791c26c7fc18a85a47da5b23f/src/modules/hyprland/window.cpp#L194-L198

**Reply by izmyname** [reply]
Can confirm - removing this code fixes waybar

**Reply by izmyname** [reply]
Would you like to create pr, btw?

**Reply by Aqa-Ib** [reply]
Ok

**Reply by izmyname** [reply]
Thank you again!

**Reply by Aqa-Ib** [reply]
https://github.com/Alexays/Waybar/pull/3747

No problem!

**Reply by izmyname** [reply]
Yey, the fix made it into master
